db/blob/blob_source.h

```diff
@@ -133,6 +154,11 @@ class BlobSource {
 
   // A cache to store uncompressed blobs.
   std::shared_ptr<Cache> blob_cache_;
+
+  // The control option of how the cache tiers will be used. Currently rocksdb
+  // support block/blob cache (volatile tier), secondary cache (non-volatile
```

Comment:
I'd say let's either remove the non-volatile part or make it clear that the name is historical (the compressed cache is in volatile memory).

Reply:
Done.